builtin/providers/google/resource_compute_forwarding_rule.go

```diff
@@ -93,13 +127,24 @@ func resourceComputeForwardingRuleCreate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	ps := d.Get("ports").(*schema.Set).List()
```

Comment:
You're totally right, and I learned some new stuff about `*ResourceData.Get` and `*schema.Set.List` today. Thanks!